packages/blockchain-wallet-v4-frontend/src/modals/BuySell/ThreeDSHandlerStripe/index.tsx

```diff
@@ -12,6 +12,7 @@ import { getData } from './selectors'
 import Success from './template.success'
 
 const ThreeDSHandlerStripe = ({ data, ...props }: Props) => {
+  // TODO pass the logic to here
```

Comment:
Still a TODO?

Reply:
yeah, but that code wasn't from me, development hadn't been updated with recently releases. I got all the branches up to date so this shouldn't be a code addition now.